web/src/components/pages/contribution/success.tsx

```diff
@@ -31,6 +31,26 @@ const GoalPercentage = ({
   </span>
 );
 
+const ContributeMoreButton = (props: {
+  hasAccount: boolean;
+  children: React.ReactNode;
+  onClick: () => any;
+}) =>
+  props.hasAccount ? (
+    <Button
+      className="contribute-more-button"
+      rounded
+      onClick={props.onClick}
```

Comment:
I think it should be props.reset

Reply:
Are you suggesting that the prop be named `reset` instead of `onClick`?

Reply:
yes, what do you think ?

Reply:
I think that `onClick` is more accurate in the scope of this component. If we called this `reset`, then we would be leaking responsibilities of the parent component down into this component. On it's own, the ContributeMoreButton is just a button, and the user of the button can listen to "onClick", but it makes less sense to listen to "reset."

Does that make sense?